### DISCUSSION TITLE
Group by 120 minutes

### DISCUSSION THREAD

**Discussion by ChrizZz90** [marcusolsson/grafana-hourly-heatmap-panel#54] [Ideas]
I thought it is easy and I just have to add a new option, but if I do so, the plugin disappears after a Grafana restart. If it is just adding a new option I am happy to create a pull request. Don't know if Grafana has a conflict if I make changes manually.

Maybe one additional question: If I have 2 overlapping entries, how would they be displayed? Next to each other or grouped?

**Comment by marcusolsson** [comment]
I don't mind adding 120 minutes as well if you submit a pull request. Seems reasonable to me 👍 

I'm not sure what you mean by "duplicate entries" though?

**Comment by ChrizZz90** [comment]
I mean I want to display timeslots and some of them are overlapping, like 09.00-11.00 and 10,00-12.00. In my understanding, they should be in the same column of the day just beside each other and overlap 1h (10.00-11.00).

**Comment by marcusolsson** [comment]
Could you post an example? There shouldn't be any overlapping time slots. For 120 minute buckets, there should be 12 slots per day, i.e. 00:00–02:00, 02:00–04:00, 04:00–06:00, and so on.
